packages/genesys-spark-components/src/components/stable/gux-dropdown-multi/gux-dropdown-multi.scss

```diff
@@ -142,8 +142,7 @@
     var(--gse-ui-formControl-input-active-border-color);
   border-radius: var(--gse-ui-formControl-input-borderRadius);
 
-  &:focus-visible,
-  &:focus-within:has(:focus-visible) {
+  &:focus-visible {
```

Comment:
Would something like 
```
  &:focus-visible {
    @include focus.gux-focus-ring;
  }

  &:focus-within:has(:focus-visible) {
    @include focus.gux-focus-ring;
  }
```
be a smaller change?

I think we would not be changing anything in chrome at all and the failing firefox section would not affect the ` &:focus-visible` code anymore.

Reply:
Yeah I think that would work. Though I tested all 3 browsers after I removed it entirely and didn't see any difference in Safari and Chrome

Reply:
🤷 maybe it is just redundant code so. I just just looking at the PR in isolation and trying to make the minimum change possible. Whichever approach you want to go with I am fine with if it does not change any behaviours in non firefox browsers.

Reply:
I did it your way just in case I missed something